.github/labeler.yml

```diff
@@ -30,7 +30,6 @@ INFRA:
   - dev/**/*
 BUILD:
   - "**/*gradle*"
-  - versions.props
```

Comment:
Should this be replaced with the version catalog file?

Reply:
I've just moved toml file to gradle dir, now it should be a part of line 32 for labeler and we don't need to explicitly add it here.